digdag-server/src/main/java/io/digdag/server/rs/AttemptResource.java

```diff
@@ -302,7 +302,16 @@ public Response startAttempt(RestSessionAttemptRequest request)
                 .getSessionStore(getSiteId())
                 .getTasksOfAttempt(attemptId);
 
+        List<Long> groupRetryErrorTaskIds = tasks.stream()
+                .filter(task -> task.getState() == TaskStateCode.GROUP_ERROR && task.getStateParams().has("retry_count"))
```

Comment:
May I ask you the purpose to check "retry_count" ?

Reply:
Yes, it's to determine if a group retry has occurred or not.
Whenever a group retry occurs, there is a `retry_count` in stateParams.

Reply:
Got it. Thanks for your explanation.

Reply:
Using `retry_count` as the switch of group retry seems a bit weird to me (Sorry if we've already discussed it. I may have had a wrong idea) since I thought `retry_count` is a just retry count for unexpected failures not like affecting the graph of tasks.